src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -656,6 +656,13 @@ while bRunning do
                     table.remove( tLines, y )
                     setCursor( sPrevLen + 1, y - 1 )
                     redrawText()
+                elseif y == 1 and type( tLines[2] ) == "string" then
+                    -- Remove first line
+                    local sPrevLen = string.len( tLines[2] )
+                    tLines[2] = tLines[2] .. tLines[1]
```

Comment:
Soo if i read this operation right then by pressing backspace at beginning of the first line you would cause contents of first line to be added to end of second line and then first line is removed. Is that something that exists in some text editor? Personally i never saw one acting like that.